lib/shared/envs.ts

```diff
@@ -1,2 +1,2 @@
-export const test = "https://test.stytch.com/v1/";
-export const live = "https://api.stytch.com/v1/";
+export const test = "https://test.stytch.com/";
+export const live = "https://api.stytch.com/";
```

Comment:
Why is `v1` being removed?

Reply:
It's part of the overall HTTP route, so the "base" is being simplified